psi-ms.obo

```diff
@@ -22920,6 +22920,166 @@ relationship: has_optional_column MS:1001844 ! MS1 feature area
+
+[Term]
+id: MS:4000081
+name: 1st PC 
+def: Data from the first principal component of a PCA. [] 
+relationship: has_value_concept NCIT:C60694 ! Principal Component 
+is_a: MS:4000080 ! QC non-metric term
+
+[Term]
+id: MS:4000082
+name: 2nd PC 
+def: Data from the second principal component of a PCA. [] 
+relationship: has_value_concept NCIT:C60694 ! Principal Component 
+is_a: MS:4000080 ! QC non-metric term
+
+[Term]
+id: MS:4000083
+name: 3rd PC 
+def: Data from the third principal component of a PCA. [] 
+relationship: has_value_concept NCIT:C60694 ! Principal Component 
+is_a: MS:4000080 ! QC non-metric term
+
+[Term]
+id: MS:4000084
+name: 4th PC 
+def: Data from the fourth principal component of a PCA. [] 
+relationship: has_value_concept NCIT:C60694 ! Principal Component 
+is_a: MS:4000080 ! QC non-metric term
+
+[Term]
+id: MS:4000085
+name: 5th PC 
+def: Data from the fifth principal component of a PCA. [] 
+relationship: has_value_concept NCIT:C60694 ! Principal Component 
+is_a: MS:4000080 ! QC non-metric term
+
+[Term]
+id: MS:4000086
+name: mzQC input reference 
+def: Used to refer to data elements of input sections in mzQC, either inputFile names or metadata labels. [] 
+is_a: MS:4000080 ! QC non-metric term
+relationship: has_value_type xsd\:string ! The allowed value-type for this CV term
+
+[Term]
+id: MS:4000087
+name: mzQC plot label 
+def: Used to supply alternative labels for plotting figures. [] 
+is_a: MS:4000080 ! QC non-metric term
+relationship: has_value_type xsd\:string ! The allowed value-type for this CV term
+
+[Term]
+id: MS:4000088
+name: batch label
+def: Used to supply batch label information with any string value. []
+is_a: MS:4000080 ! QC non-metric term
+relationship: has_value_type xsd\:string ! The allowed value-type for this CV term
+	
+[Term]
+id: MS:4000089
+name: injection sequence label
+def: Used to supply injection sequence information with consecutive whole numbers. []
+is_a: MS:4000080 ! QC non-metric term
+relationship: has_value_type xsd\:int ! The allowed value-type for this CV term
+
+[Term]
+id: MS:4000090
+name: PCA of MaxQuant's protein group raw intensities
+def: A table with the PCA results of MaxQuant's protein group raw intensities. []
+is_a: MS:4000005 ! table
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_metric_category MS:4000021 ! MS1 metric
+relationship: has_metric_category MS:4000013 ! multiple runs based metric
+relationship: has_column MS:4000081 ! 1st PC
+relationship: has_column MS:4000086 ! mzQC input reference 
+relationship: has_optional_column MS:4000082 ! 2nd PC 
+relationship: has_optional_column MS:4000083 ! 3rd PC 
+relationship: has_optional_column MS:4000084 ! 4th PC 
+relationship: has_optional_column MS:4000085 ! 5th PC 
+relationship: has_optional_column MS:4000087 ! mzQC plot label 
+relationship: has_optional_column MS:4000088 ! batch label 
+relationship: has_optional_column MS:4000089 ! injection sequence label
```

Comment:
I need it for the plot generated for https://github.com/HUPO-PSI/mzQC/blob/main/doc/figures/MetaboPCAbefore.png 
and I think it will be interesting for whole experiment QA

Reply:
But if you won't need these bcs' for example MQ does not expose such annotation anyways, I'll remove it?

Reply:
Do definitions fit otherwise?

Reply:
> But if you won't need these bcs' for example MQ does not expose such annotation anyways, I'll remove it?

Maxquant does not support those. So +1 for remove

> Do definitions fit otherwise?

Yes. looks good